src/app/job/[id]/page.module.css

```
@@ -0,0 +1,72 @@
+.lists {
+  display: grid;
+  gap: 1rem;
+  padding: 1rem;
+  border: 1px solid transparent;
+  border-bottom-color: var(--neutral-shade-90);
+
+  &:hover {
+    border-color: var(--neutral-shade-60);
+    border-radius: var(--border-radius-square);
+    background-color: var(--primary-tint-02);
+  }
+
+  header {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    .company {
+      display: flex;
+      align-items: center;
+      gap: 0.5rem;
+      .company-image Image {
+        min-block-size: 100%;
+        block-size: 0;
+
+        Image {
+          block-size: 100%;
+          inline-size: auto;
+          aspect-ratio: 1 / 1;
+          object-fit: cover;
+          border: 1px solid var(--neutral-shade-50);
+          border-radius: var(--border-radius-circle);
+        }
+      }
+
+      span {
+        color: var(--neutral-shade-60);
+        text-align: left;
```

بهتره از Logical Values استفاده کنید

```suggestion
        text-align: start;
```

---

هرجایی که از این روش رفتم را اصلاح کردم